girder/constants.py

```diff
@@ -195,6 +195,12 @@ class TokenScope:
     ANONYMOUS_SESSION = 'core.anonymous_session'
     USER_AUTH = 'core.user_auth'
     TEMPORARY_USER_AUTH = 'core.user_auth.temporary'
+    PLUGINS_ENABLED_READ = 'core.plugins.read'
```

Comment:
Well, you know how it is, as soon as you commit something, we're stuck supporting it forever, even if it was a bad idea. However, if you're comfortable with this list as a base (I assume you've put some thought into it), that's fine with me.